gaphor/UML/classes/interface.py

```diff
@@ -158,8 +158,8 @@ class InterfaceItem(ElementPresentation, Classified):
     RADIUS_REQUIRED = 14
 
     def __init__(self, connections, id=None, model=None):
-        super().__init__(connections, id, model)
         self._folded = Folded.NONE
+        super().__init__(connections, id, model)
```

Comment:
Did you mean to swap the super class constructor so that it is below the assignment here?

Reply:
Yes, it's on purpose. `update_shapes()` is called from `ElementPresentation`'s constructor, and it requires this attribute.

Reply:
Ahh got it, i knew there had to be a reason for it, but it wasn't immediately clear